src/soep_preparation/utilities/general.py

```diff
@@ -5,18 +5,38 @@
 from importlib.util import module_from_spec, spec_from_file_location
 from pathlib import Path
 from types import ModuleType
+from typing import Any
 
 
 def _fail_if_raw_data_file_missing(
     script_name: str, data_root: Path, soep_version: str
 ) -> None:
     raw_data_file_path = data_root / f"{soep_version}" / f"{script_name}.dta"
     if not raw_data_file_path.exists():
-        msg = f"""Raw data file {raw_data_file_path} not found for SOEP {soep_version}.
-        Ensure the file is present in the data directory for the corresponding wave."""
+        msg = (
+            f"Raw data file {raw_data_file_path} not found for SOEP {soep_version}.\n"
+            f"Ensure the file is present in the data directory\n"
+            f" corresponding to the specified wave."
+        )
         raise FileNotFoundError(msg)
 
 
+def get_variable_names_in_script(script: Any) -> list[str]:  # noqa: ANN401
```

Comment:
This is a bit confusing for me. For one, as an ex-Stata user, "`variable_names`" make me think of "column names" in a data project. So maybe "`global_object_names`" ? Second, the function name is very general, but then it only returns those objects that start with `derive_`?